### PR TITLE
Config cloudinary

### PR DESCRIPTION
**Pull request by celinelemenn**

### CODE REVIEW

File: app/uploaders/photo_uploader.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  include Cloudinary::CarrierWave
+
+  process eager: true # Force version generation at upload time.
+
+  process convert: 'jpg'
+
+  version :thumnail do
+    resize_to_fit 256, 256
+  end
+
```
Comment (AP2610) [inline app/uploaders/photo_uploader.rb]: `version :avatar do
    cloudinary_transformation transformation: [
      { width: 400, height: 400, gravity: :face, radius: :max, crop: :thumb },
      { width: 200, height: 200, crop: :thumb }
    ]
  end`

File: app/uploaders/photo_uploader.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  version :bright_face do
+    cloudinary_transformation effect: "brightness:30", radius: 20,
+      width: 150, height: 150, crop: :thumb, gravity: :face
+  end
+
+end
+
```
Comment (AP2610) [inline app/uploaders/photo_uploader.rb]: We can delete everything below this line i believe

File: app/models/user.rb [hunk 3]
```diff
@@ -14,4 +14,6 @@ class User < ApplicationRecord
   has_many :comments
 
   has_many :user_positions
+
+  mount_uploader :remote_photo_url, PhotoUploader
```
Comment (AP2610) [inline app/models/user.rb]: it should be changed to :photo instead of :remote_photo_url

File: app/uploaders/photo_uploader.rb (hunk 1 above)
Comment (celinelemenn) [reply app/uploaders/photo_uploader.rb]: added in the code

File: app/uploaders/photo_uploader.rb (hunk 2 above)
Comment (celinelemenn) [reply app/uploaders/photo_uploader.rb]: keeping just as a reference if we do more things later with cloudinary

File: app/models/user.rb (hunk 3 above)
Comment (celinelemenn) [reply app/models/user.rb]: user story made for this refactoring

**Pull request merged**